src/config/validate/runtime.ts

```diff
@@ -41,7 +41,7 @@ const validDistributionShares = Joi.custom((i) => {
   }
 });
 
-const validRpcUrl = Joi.string().uri({ scheme: ["https"] });
+const validRpcUrl = Joi.string().uri({ scheme: ["https", "http"] });
```

Comment:
Technically any protocol is ok if we run in local setup. So we can add tcp here as well.